### PR TITLE
Adding the username program

### PR DESCRIPTION
**Pull request by seidenstein**

### CODE REVIEW

File: username.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+def minLen(strP):
+  print("minLen")
+  return (len(strP)>3)
```
Comment (dbtleonia) [inline username.py]: Remove parens around this expression.

File: username.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+def maxLen(strP):
+  print ("maxLen")
```
Comment (dbtleonia) [inline username.py]: Remove space after print.

File: username.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+def minLen(strP):
+  print("minLen")
+  return (len(strP)>3)
+def maxLen(strP):
```
Comment (dbtleonia) [inline username.py:5]: Add a blank line between functions.

**Review by dbtleonia — COMMENTED** [review]
Some initial comments.

**Review by seidenstein — COMMENTED** [review]
All comments addressed. PTAL

**Review by dbtleonia — APPROVED** [review]
LGTM

**Pull request merged**